Tasks/Common/AzureHelpers/InitializeFunctions.ps1

```diff
@@ -1,37 +1,21 @@
 ﻿$script:isClassic = $null
 $script:classicVersion = $null
 
-function Set-CurrentAzureSubscription {
-    [CmdletBinding()]
-    param(
-        [Parameter(Mandatory=$true)]
-        [string]$SubscriptionId,
-        [string]$StorageAccount)
-
-    $additional = @{ }
-    if ($script:isClassic -and $script:classicVersion -lt ([version]'0.8.15')) {
-        $additional['Default'] = $true # The Default switch is required prior to 0.8.15.
-    }
-
-    Write-Host "##[command]Select-AzureSubscription -SubscriptionId $SubscriptionId $(Format-Splat $additional)"
-    $null = Select-AzureSubscription -SubscriptionId $SubscriptionId @additional
-    if ($StorageAccount) {
-        Write-Host "##[command]Set-AzureSubscription -SubscriptionId $SubscriptionId -CurrentStorageAccountName $StorageAccount"
-        Set-AzureSubscription -SubscriptionId $SubscriptionId -CurrentStorageAccountName $StorageAccount
-    }
-}
-
-function Set-CurrentAzureRMSubscription {
+function Add-Certificate {
```

Comment:
I moved this certificate logic (not unit testable in it's current form) into a private function so I could unit test the caller of this function.